packages/common-types/src/devices.ts

```diff
@@ -18,7 +18,7 @@ export interface IosSimulator {
   runtime: string;
   osVersion: string;
   windowName: string;
-  osType: "iOS";
+  osType: "iOS" | "tvOS";
```

Comment:
having this inside `IosSimulator` sounds a little bit weird to me, maybe we should have a "base" simulator type and `IosSimulator` and `tvOsSimulator`

Reply:
or maybe calling this `AppleSimulator`, what do you think?

Reply:
I like `AppleSimulator` 👍

Reply:
I'll do the `AppleSimulator` change in a separate PR